bindings/ergo-lib-wasm/src/ergo_box/box_builder.rs

```diff
@@ -112,8 +112,9 @@ impl ErgoBoxCandidateBuilder {
     }
 
     /// Build the box candidate
-    pub fn build(self) -> Result<ErgoBoxCandidate, JsValue> {
+    pub fn build(&self) -> Result<ErgoBoxCandidate, JsValue> {
```

Comment:
FYI these used to destroy the builder, but after this PR the old object is kept in memory until you `free` it explicitly. This is useful for us in Yoroi because the way the wallet works is that the transaction confirmation page actually populates the UI directly from a TxBuilder. It called the `build()` once to populate the UI (to show the confirmation page), then called it again later to actually sign the transaction. It's much easier to keep the builder in memory than keeping the `UnsignedTransaction` in memory since the builder retains more information about how the tx was built (ex: which addresses were change, etc.)